lib/brew/gem/formula.rb.erb

```diff
@@ -61,8 +61,9 @@ class <%= klass %> < Formula
                          ].first
     zsh_completion.install completion_for_zsh if completion_for_zsh
 
+    bindir = Gem::Specification::load("#{prefix}/specifications/<%= name %>-#{version}.gemspec").bindir
     ruby_libs = Dir.glob("#{prefix}/gems/*/lib")
-    Pathname.glob("#{brew_gem_prefix}/bin/*").each do |file|
+    Pathname.glob("#{brew_gem_prefix}/#{bindir}/*").each do |file|
```

Comment:
This looks great. One suggestion as long as we're here. A `Gem::Specification` also has an `executables` field that specifies which scripts should be executable. How about instead of globbing the bindir that we do something like

```
gemspec = Gem::Specification::load(...)
bindir = gemspec.bindir
ruby_libs = Dir.glob("#{prefix}/gems/*/lib")
executables = gemspec.executables.map {|exe| "#{brew_gem_prefix}/#{bindir}/#{exe}" }
executables.each do |file|
  # rest of the code below
end
```